### PR TITLE
Fixes to work with vagrant locally for develop environment

### PR DESCRIPTION
**Pull request by xcompass**

### CODE REVIEW

File: tasks/tomcat.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@
     port: 8080/tcp
     permanent: true
     immediate: true
-    state: "{{ dspace_use_proxy | ternary('disabled', 'enabled') }}"
+    state: "{{ (dspace_use_proxy == 'true') | ternary('disabled', 'enabled') }}"
```
Comment (cubranic) [inline tasks/tomcat.yml]: I don't know enough Jinja to understand this: was not having `== 'true'` not working right?

Also, with YAML, I see also `yes` being used interchangeably with `true` (and `True`, etc.): [Ansible docs](http://docs.ansible.com/ansible/YAMLSyntax.html). Would this comparison still work?

Comment (cubranic) [inline tasks/tomcat.yml]: On a little extra reading, it looks like `dspace_use_proxy | bool | ternary(...)` might be the right thing.

Comment (xcompass) [inline tasks/tomcat.yml]: Maybe . I'll give a try. I thin dspace_use_proxy is a string.

**Comment by cubranic** [comment]
@xcompass Can you move the site configuration to your fork of this repo? It's been over a year since I was involved in the project and I'd like to remove it.

**Comment by xcompass** [comment]
Sorry, it was because of this open PR. The change made on my repo updated in this PR. I'm closing this PR now.

**Pull request closed without merging**